README.md

```diff
@@ -1,2 +1,167 @@
-# template
-Use this template for web standards proposals in this org.
+# Explainer for the TODO API
+
+**Instructions for the explainer author: Search for "todo" in this repository and update all the
+instances as appropriate. For the instances in `index.bs`, update the repository name, but you can
+leave the rest until you start the specification. Then delete this block of text.**
+
+This proposal is an early design sketch by [TODO: team] to describe the problem below and solicit
+feedback on the proposed solution. It has not been approved to ship in Chrome.
+
+## Proponents
+
+- [Proponent team 1]
+- [Proponent team 2]
+- [etc.]
+
+## Participate
+- [Issue tracker]
+- [Discussion forum]
+
+## Table of Contents [if the explainer is longer than one printed page]
+
+<!-- Update this table of contents by running `npx doctoc README.md` -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Goals [or Motivating Use Cases, or Scenarios]](#goals-or-motivating-use-cases-or-scenarios)
+- [Non-goals](#non-goals)
+- [User research](#user-research)
+- [[API 1]](#api-1)
+- [[API 2]](#api-2)
+- [Key scenarios](#key-scenarios)
+  - [Scenario 1](#scenario-1)
+  - [Scenario 2](#scenario-2)
+- [Detailed design discussion](#detailed-design-discussion)
+  - [[Tricky design choice #1]](#tricky-design-choice-1)
+  - [[Tricky design choice 2]](#tricky-design-choice-2)
+- [Considered alternatives](#considered-alternatives)
+  - [[Alternative 1]](#alternative-1)
+  - [[Alternative 2]](#alternative-2)
+- [Stakeholder Feedback / Opposition](#stakeholder-feedback--opposition)
+- [References & acknowledgements](#references--acknowledgements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction
+
+TODO: Fill this in using https://tag.w3.org/explainers/ as a reference.
+
+[The "executive summary" or "abstract".
+Explain in a few sentences what the goals of the project are,
+and a brief overview of how the solution works.
+This should be no more than 1-2 paragraphs.]
+
+## Goals [or Motivating Use Cases, or Scenarios]
```

Comment:
Not clear how this interacts with the Key Scenarios below?  Definitely need to make sure htis gets filled out, in depth, prior to the potential solution.

Reply:
Yeah. I took this directly from the [TAG's template](https://github.com/w3ctag/tag.w3.org/blob/main/explainers/template.md), but it's not clear. I think the idea was to list use cases here, and then have the Key Scenarios explain how the potential solutions solve the use cases. I'll try to make that clearer ... and then also send the TAG a patch.